core/src/mindustry/world/blocks/experimental/BlockForge.java

```diff
@@ -18,13 +19,16 @@
 import mindustry.world.blocks.production.*;
 import mindustry.world.consumers.*;
 
+import static mindustry.Vars.*;
+
 public class BlockForge extends PayloadAcceptor{
     public float buildSpeed = 0.4f;
+    public int minBlockSize = 1;
+    public int maxBlockSize = 2;
 
     public BlockForge(String name){
         super(name);
 
-        size = 3;
```

Comment:
I don't see a good reason to remove this; block forges should still be 3x3 by default, as anything smaller wouldn't be particularly useful - I'm not sure if it would even work with the payload system. Likewise with block loaders.

Reply:
It allows you to make *bigger* block forges(T2 or higher) that will craft larger blocks.

> I'm not sure if it would even work with the payload system.

Well it is experimental, and besides, in vanilla the only versions will be the 3x3 ones. higher ones are for mods!

Reply:
I'l commit follow-up PRs if the higher ones misbehave. Also by the same line, you also allow 2x2 or higher conveyors, despite them never working.

Reply:
> It allows you to make bigger block forges(T2 or higher) that will craft larger blocks.

It changes nothing. Super-constructors run before any user code can interact with the object.

Reply:
> It changes nothing. Super-constructors run before any user code can interact with the object.
I literally do not know what that means but ok